### PR TITLE
Storage fast check contains

### PR DESCRIPTION
**Pull request by pyakushin**

### CODE REVIEW

File: src/storage/core.rs [hunk 1]
```diff
@@ -412,17 +412,38 @@ impl<K> Storage<K> {
...
+        in_active
+    }
+
+    /// `fast_check` is used to check whether a key is in storage.
     /// Uses bloom filter under the hood, so false positive results are possible,
     /// but false negatives are not.
-    /// In other words, `contains` returns either "possibly in storage" or "definitely not".
-    pub async fn contains(&self, key: impl Key) -> bool {
+    /// In other words, `fast_check` returns either "possibly in storage" or "definitely not".
+    pub async fn fast_check(&self, key: impl Key) -> bool {
```
Comment (agend) [inline src/storage/core.rs]: if this is bloom filter - lets call this method accordingly (`check_bloom`). In pearl as well please)

File: src/blob/core.rs [hunk 2]
```diff
@@ -191,8 +191,8 @@ impl Blob {
         self.index.get_entry(key, self.file.clone())
     }
 
-    async fn lookup(&self, key: &[u8], meta: Option<&Meta>) -> Option<Location> {
-        if self.contains(key) {
+    async fn get_location(&self, key: &[u8], meta: Option<&Meta>) -> Option<Location> {
```
Comment (agend) [inline src/blob/core.rs]: why did you change from lookup to get_location?

Comment (pyakushin) [reply src/blob/core.rs]: No particular reason, thought it would be more clear
